runtime/base/base.Dockerfile

```diff
@@ -49,11 +92,11 @@ RUN mkdir -p ${BUILD_DIR}  \
 # https://github.com/madler/zlib/releases
 # Needed for:
 #   - openssl
+#   - curl
 #   - php
 # Used By:
 #   - xml2
-ARG zlib
-ENV VERSION_ZLIB=${zlib}
+ENV VERSION_ZLIB=1.2.11
```

Comment:
I inlined all version numbers. The `versions.ini` + script (now removed) made things harder to understand for me.